netman/adapters/switches/brocade.py

```diff
@@ -463,6 +472,8 @@ def add_interface_vlan_data(target_vlan, int_vlan_data):
             vrrp_group = None
         elif regex.match("^ ip helper-address ([^\s]*)", line):
             target_vlan.dhcp_relay_servers.append(IPAddress(regex[0]))
+        elif regex.match("^\s*no ip redirect", line):
```

Comment:
1 space. The change in fake switch was to match the exact number of white space the real brocade prints out. 
I didn't think it was big enough of a deal to create the tag in fake switch and match with the exact number of space.
But if you think it would be nicer I can do it.